### PR TITLE
create b2b tests for token endpoints

### PR DESCRIPTION
**Pull request by m-alama**
## PR Description

create b2b tests for token

### CODE REVIEW

File: tests/api/b2b/glue/access_token_endpoints/token/positive.robot [hunk 1]
```diff
@@ -0,0 +1,16 @@
+*** Settings ***
+Suite Setup       SuiteSetup
+Resource    ../../../../../../resources/common/common_api.robot
+
+*** Test Cases ***
+
+Get_token_for_customer
```
Comment (OKrivtsova) [inline tests/api/b2b/glue/access_token_endpoints/token/positive.robot:7]: we also need a test for refresh token (positive and negative)
see here https://spryker.atlassian.net/wiki/spaces/DOCS/pages/2449047998/Published+Managing+customer+authentication+tokens+via+OAuth+2.0

File: tests/api/b2b/glue/access_token_endpoints/token/negative.robot [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+Get_token_for_customer_using_invalid_grant_type
+    When I set Headers:    Content-Type=${urlencoded_header_content_type}
+    Then I send a POST request with data:    /token    {"grant_type": "invalid_grant_type","username": "${yves_user_email}","password": "${yves_user_password}"}
+    And Response status code should be:    400
+    And Response body parameter should be:    [error]    invalid_grant
+    And Response body parameter should be:    [error_description]    The provided authorization grant (e.g., authorization code, resource owner credentials) or refresh token is invalid, expired, revoked, does not match the redirection URI used in the authorization request, or was issued to another client.
+
+
+
+Get_token_for_customer_usin_wrong_password
```
Comment (OKrivtsova) [inline tests/api/b2b/glue/access_token_endpoints/token/negative.robot:16]: we also need test for customer that does not exist, as well as for missing and empty required body parameters

File: tests/api/b2b/glue/access_token_endpoints/token/negative.robot [hunk 3]
```diff
@@ -0,0 +1,21 @@
+*** Settings ***
+Suite Setup       SuiteSetup
+Resource    ../../../../../../resources/common/common_api.robot
```
Comment (OKrivtsova) [inline tests/api/b2b/glue/access_token_endpoints/token/negative.robot:3]: please add these to settings and add them in future tests
*** Settings ***

Test Setup TestSetup
Default Tags glue

File: tests/api/b2b/glue/access_token_endpoints/token/positive.robot [hunk 4]
```diff
@@ -0,0 +1,16 @@
+*** Settings ***
+Suite Setup       SuiteSetup
+Resource    ../../../../../../resources/common/common_api.robot
```
Comment (OKrivtsova) [inline tests/api/b2b/glue/access_token_endpoints/token/positive.robot:3]: please add these to settings and add them in future tests
*** Settings ***

Test Setup TestSetup
Default Tags glue

Comment (m-alama) [reply tests/api/b2b/glue/access_token_endpoints/token/positive.robot:4]: Done

File: tests/api/b2b/glue/access_token_endpoints/token/negative.robot (hunk 3 above)
Comment (m-alama) [reply tests/api/b2b/glue/access_token_endpoints/token/negative.robot:4]: Done

File: tests/api/b2b/glue/access_token_endpoints/token/negative.robot (hunk 2 above)
Comment (m-alama) [reply tests/api/b2b/glue/access_token_endpoints/token/negative.robot]: Done

File: tests/api/b2b/glue/access_token_endpoints/token/positive.robot (hunk 1 above)
Comment (m-alama) [reply tests/api/b2b/glue/access_token_endpoints/token/positive.robot:9]: Done

**Review by OKrivtsova — APPROVED** [review]
